manager/state/raft/raft.go

```diff
@@ -133,10 +134,11 @@ type Node struct {
 	// to stop.
 	stopped chan struct{}
 
-	lastSendToMember map[uint64]chan struct{}
-	raftLogger       *storage.EncryptedRaftLogger
-	keyRotator       EncryptionKeyRotator
-	rotationQueued   bool
+	lastSendToMember    map[uint64]chan struct{}
+	raftLogger          *storage.EncryptedRaftLogger
+	keyRotator          EncryptionKeyRotator
+	rotationQueued      bool
+	waitForAppliedIndex *uint64
```

Comment:
1 is the lowest index. I see the etcd/raft code using 0 to represent unset (`Index` is compared against `None`, which is a constant value set to 0).